minecraft_servers/warfaremc/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "server_name": "warfaremc",
+  "nice_name": "WarfareMC",
+  "direct_ip": "mc.warfaremc.eu",
+  "server_wildcards": [
+    "play.warfaremc.eu"
+  ],
+  "supported_languages": [
+    "cz",
+    "sk"
+  ],
+  "social": {
+    "web": "https://warfaremc.eu",
+    "web_shop": "https://shop.warfaremc.eu",
+    "twitter": "warfaremceu",
+    "discord": "https://discord.gg/C3Vn4aY",
+    "tiktok": "warfaremc",
+	"instagram": "warfaremceu",
```

Comment:
fix formatting

Reply:
Should be fixed, idk how i really tabbed there :S